src/js/dom.js

```diff
@@ -0,0 +1,163 @@
+      const h5 = helpers().createElement('h5', 'card-title text-center text-warning');
+      h5.innerHTML = day.dt_txt.slice(0, -9);
+      const listCont = helpers().createElement('div', 'd-flex flex-row justify-content-around');
+      const ul1 = helpers().createElement('ul', 'list-group list-unstyled');
+      const ul2 = helpers().createElement('ul', 'list-group list-unstyled');
+      const temp = helpers().createElementWithInnerText('li', 'list-group-item', `<i class="fas fa-thermometer-half text-warning my-2"></i> Temp: ${day.main.temp} Celsius`);
+      const feel = helpers().createElementWithInnerText('li', 'list-group-item', `<i class="fas fa-meteor text-warning my-2"></i> Feel: ${day.main.feels_like} Celsius`);
+      const pressure = helpers().createElementWithInnerText('li', 'list-group-item', `<i class="fas fa-compress-arrows-alt text-warning my-2"></i> Pressure: ${day.main.pressure} hPa`);
+      const humidity = helpers().createElementWithInnerText('li', 'list-group-item', `<i class="fas fa-percent text-warning my-2"></i> Humidity: ${day.main.humidity}%`);
+      const desc = helpers().createElementWithInnerText('li', 'list-group-item', `<i class="fas fa-cloud-sun-rain text-warning my-2"></i> ${day.weather[0].main}`);
+      const minTemp = helpers().createElementWithInnerText('li', 'list-group-item', `<i class="fas fa-temperature-low text-warning my-2"></i> Min: ${day.main.temp_min} Celsius`);
+      const maxTemp = helpers().createElementWithInnerText('li', 'list-group-item', `<i class="fas fa-temperature-high text-warning my-2"></i> Max: ${day.main.temp_max} Celsius`);
+      const clouds = helpers().createElementWithInnerText('li', 'list-group-item', `<i class="fas fa-cloud text-warning my-2"></i> Clouds: ${day.clouds.all}%`);
+      const wind = helpers().createElementWithInnerText('li', 'list-group-item', `<i class="fas fa-wind text-warning my-2"></i> ${day.wind.speed} meter/sec`);
+      const windDir = helpers().createElementWithInnerText('li', 'list-group-item', `<i class="fas fa-compass text-warning my-2"></i> ${day.wind.deg} degrees`);
+      const color = document.getElementsByClassName('list-group-item');
+
+      ul2.appendChild(minTemp);
+      ul2.appendChild(maxTemp);
+      ul2.appendChild(wind);
+      ul2.appendChild(windDir);
+      ul2.appendChild(clouds);
+
+      ul1.appendChild(temp);
+      ul1.appendChild(feel);
+      ul1.appendChild(desc);
+      ul1.appendChild(pressure);
+      ul1.appendChild(humidity);
+
+      listCont.appendChild(ul1);
+      listCont.appendChild(ul2);
+
+      over.appendChild(h5);
+      over.appendChild(listCont);
+
+      card.appendChild(imgBg);
+      card.appendChild(over);
+
+      cont.appendChild(card);
+      row.appendChild(cont);
+
+      for (let i = 0; i < color.length; i++) {
+        color[i].style = 'background-color: rgba(255,255,255,0.1);';
+      }
+
+      // eslint-disable-next-line no-useless-concat
+      imageSwitch(day, ('images' + `${index + 1}`));
```

Comment:
Technically template strings are used to avoid concatenations with + the above code would work like this 
`imageSwitch(day, (`images ${index + 1}`));`